docs/poc2023q1/example1.md

```diff
@@ -0,0 +1,256 @@
+
+Stage 1 has the following steps.
+
+1. Create two kind clusters.
+    ```shell
+    $ kind create cluster kind1
+    $ kind create cluster kind3
+    ```
+
+2. Start `kcp`.  In the following shell commands it is assumed that
+   `kcp` is running and `$KUBECONFIG` is set to the
+   `.kcp/admin.kubeconfig` that `kcp` produces.
+
+3. Create an inventory management workspace.
+    ```shell
+    $ kcp ws root
+    $ kcp ws create inv1 --enter
+    ```
+
+4. Create a SyncTarget object to represent the kind1 cluster.
+    ```shell
+    $ kubectl create -f - <<<EOF
+    apiVersion: workload.kcp.io/v1alpha1
+    kind: SyncTarget
+    metadata:
+      name: kind1
+      labels:
+        example: yes
+        extended: no
+    spec:
+      cells:
+        foo: bar
+    EOF
+    ```
+
+5. Create a Location object describing the kind1 cluster.
+    ```shell
+    $ kubectl create -f - <<<EOF
+    apiVersion: scheduling.kcp.io/v1alpha1
+    kind: Location
+    metadata:
+      name: kind1
+      annotations:
+        env: prod
+    spec:
+      resource: {group: workload.kcp.io, version: v1alpha1, resource: synctargets}
+      instanceSelector:
+        matchLabels: {"example":"yes", "extended":"no"}
+    EOF
+    ```
+
+6. Create a SyncTarget object describing the kind3 cluster.
+    ```shell
+    $ kubectl create -f - <<<EOF
+    apiVersion: workload.kcp.io/v1alpha1
+    kind: SyncTarget
+    metadata:
+      name: kind3
+      labels:
+        example: yes
+        extended: yes
+    spec:
+      cells:
+        bar: baz
+    EOF
+    ```
+
+7. Create a Location object describing the kind3 cluster.
+    ```shell
+    $ kubectl create -f - <<<EOF
+    apiVersion: scheduling.kcp.io/v1alpha1
+    kind: Location
+    metadata:
+      name: kind3
+      annotations:
+        env: prod
+        extended: yes
+    spec:
+      resource: {group: workload.kcp.io, version: v1alpha1, resource: synctargets}
+      instanceSelector:
+        matchLabels: {"example":"yes", "extended":"yes"}
+    EOF
+    ```
+
+8. Create the edge service provider workspace.
+    ```shell
+    $ kubectl ws root
+    $ kubectl ws create edge --enter
+    ```
+
+9. Populate the edge service provider workspace.
+    ```shell
+    $ kubectl create ...(details TBD)...
+    ```
+
+10. The mailbox controller will create two mailbox workspaces, one for
+    each SyncTarget.  Check it out.
+    ```shell
+    $ kubectl get Workspace
+    NAME                                                       TYPE        REGION   PHASE   URL                                                     AGE
+    niqdko2g2pwoadfb-mb-f99e773f-3db2-439e-8054-827c4ac55368   universal            Ready   https://192.168.58.123:6443/clusters/0ay27fcwuo2sv6ht   22s
+    niqdko2g2pwoadfb-mb-c5820696-016b-41f6-b676-d7c0ef02fc5a   universal            Ready   https://192.168.58.123:6443/clusters/dead3333beef3333   22s
+    ```
+
+## Stage 2
```

Comment:
please change the arrow colors for the connections between the ESPW and WMWSB to something other than black.  Would make it easier to follow visually.

Reply:
its not apparent as to the connection between the blue workload ws and the ESPWS.  Maybe draw the arrows around the green workload ws?

Reply:
Hopefully it is clear in the revised drawings.